.gitignore

```diff
@@ -35,3 +35,5 @@ node_modules
 .node_repl_history
 
 .env
+
+config/config.json
```

Comment:
Any reason you ignored `config.json`?

Reply:
The contains sensitive database information. Since we can't use environment variables in a json file, I thought it best to have it ignored and everyone can create their own locally. Is there a better approach ?

Reply:
Could you just inject the Node Environment in their respective scripts instead of creating a `json` file? 
Example:
  "dev": " NODE_ENV=development nodemon --exec babel-node src/index.js",

Reply:
Also Fix the merge conflicts

Reply:
I am throwing an error if `NODE_ENV` is missing as you requested. The merge conflicts are now fixed as well.